### ISSUE TITLE
[Bug]: MEGA syncs broken, Failed to sync folder: Invalid argument

### ISSUE THREAD

**Issue by qirlib** [termux/termux-packages#14097]
### Problem description

When I try to add a sync using mega-cmd, I get this error:
```
$ mega-sync /storage/emulated/0/folder /mega_folder
[API:err: 01:49:15] Failed to sync folder: Invalid argument
```
To capture log:
```
$ mega-quit && mega-cmd-server --debug-full | tee mega.log
# (in another terminal session run mega-sync, then mega-quit)
```
My edited log:
```
Request (ADD_SYNC) starting
Adding sync: /storage/emulated/0/folder vs /mega_folder
DNS cache hit for g.api.mega.co.nz (IPv4) 66.203.125.12
cs Sending    [REDACTED]
cs Received 4: [-2]
Request (ADD_SYNC) finished with error: Invalid argument
Failed to sync folder: Invalid argument
```
I found [this issue](https://github.com/meganz/MEGAcmd/issues/623) with devs saying this has something to do with mega-cmd failing to obtain a machine identifier and I can't read `/etc/machine-id` on my termux.

### What steps will reproduce the bug?

Log into MEGA account using mega-cmd, make needed folders, then run
`$ mega-sync <local-folder> <mega-folder>`

### System information

termux-info:

```
Termux Variables:
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://termux.mentality.rip/termux-main stable main
# x11-repo (sources.list.d/x11.list)
deb https://termux.mentality.rip/termux-x11 x11 main
Updatable packages:
All packages up to date
termux-tools version:
1.34.1
Android version:
11
Kernel build information:
Linux localhost 4.19.157-perf+ #1 SMP PREEMPT Fri Aug 26 11:42:12 IST 2022 aarch64 Android
```

**Comment by xtkoba** [comment]
Actually our `megacmd` package has some unfixed hardcoded paths. I submitted a PR to fix this: #14098.

To reproduce this it seems to require a login to an external service which is not available to me. The author of this issue is requested to report back whether or not this is resolved by a newer version (`1.5.1-1`) of `megacmd` package which will soon become available. Thanks.

**Comment by qirlib** [comment]
The update fixed it. Thank you.
